c2corg_ui/locale/c2corg_ui-client.pot

```diff
@@ -1704,15 +1708,15 @@ msgid "routes_taken"
 msgstr ""
 
 #: c2corg_ui/templates/i18n.html
-msgid "sep"
+msgid "see more results"
```

Comment:
Whatever the case I think. It's not really a sentence so capitalizing the first letter is not really mandatory. If a uppercasing the first letter proves needed, we can still do it later.